src/ui/NoNameFacet/NoNameFacetValues/NoNameFacetValueCheckbox.ts

```diff
@@ -0,0 +1,27 @@
+import { $$ } from '../../../utils/Dom';
+import { SVGIcons } from '../../../utils/SVGIcons';
+import { SVGDom } from '../../../utils/SVGDom';
+
+export class NoNameFacetValueCheckbox {
+  public element: HTMLElement;
+
+  constructor(private label: string) {
```

Comment:
The constructor should probably accept a `NoNameFacetValue `  instead of a label.